docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -16,6 +16,28 @@ The Alpakka Kafka 2.0 series features
 * Removal of some API which was deprecated since 1.0.0 or earlier.
 * Internal filter to avoid emitting records of revoked partitions [#946](https://github.com/akka/alpakka-kafka/issues/946) and [#992](https://github.com/akka/alpakka-kafka/issues/992)
 
+# 2.0.7
+
+Released: 2021-02-03
+
+## Notable changes since 2.0.6
+
+- Increase default cluster start timeouts and make configurable (backport) [#1311](https://github.com/akka/alpakka-kafka/issues/1311) by [@seglo](https://github.com/seglo)
+- Use restart script [#1310](https://github.com/akka/alpakka-kafka/issues/1310) by [@seglo](https://github.com/seglo)
```

Comment:
No need to list these twice (as the release drafter does).
Maybe a note that this release only affects the testkit?